module/applications/actor/character-sheet-2.mjs

```diff
@@ -355,7 +355,7 @@ export default class ActorSheet5eCharacter2 extends ActorSheet5eCharacter {
         name, reference,
         id: k,
         icon: img ?? icon,
-        disabled: existing ? disabled : !this.actor.statuses.has(k)
```

Comment:
Rendering conditions as enabled that are only implied by another is confusing because it gives the impression that the implied conditions can be removed. Perhaps we should have a different section on the sheet where we show the list of all status effects (not only RAW conditions) (plus encumbrance statuses) that are affecting the actor.